app/src/main/java/org/dicekeys/app/data/BackupRecipes.kt

```diff
@@ -1,10 +1,32 @@
 package org.dicekeys.app.data
 
+import kotlinx.datetime.Clock
+import kotlinx.datetime.Instant
+import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
 import org.dicekeys.api.DerivationRecipe
 
 @Serializable
-data class BackupRecipes(
-    val version: Int,
-    val recipes: List<DerivationRecipe>
-)
\ No newline at end of file
+data class BackupRecipes constructor(
+    @SerialName("version")
+    val version: Int? = null,
+
+    @SerialName("createdAt")
+    val createdAt: Instant? = null,
+
+    @SerialName("recipes")
+    val recipes: List<DerivationRecipe>,
+) {
+
+    override fun toString(): String = Json.encodeToString(this)
+
+    companion object {
+        fun create(recipes: List<DerivationRecipe>) = BackupRecipes(
+                version = 1,
+                createdAt = Clock.System.now(),
+                recipes = recipes
+            )
+    }
+}
```

Comment:
@UppaJung This is the proposed backup file.

BackupManager supports also the previous version (array of recipes).